.tests-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+echo "" > coverage.txt
```

Comment:
does `coverage.txt` needs to be created before continuing?
if not, `rm -f` it. if so, either `echo "" > coverage.txt` or `rm -f coverage.txt; touch coverage.txt` would work.